### ISSUE TITLE
Traefik 2.3.0-rc2 - MetaDatas can't be retrieved for AWS ECS Fargate

### ISSUE THREAD

**Issue by teddy-ferdinand** [containous/traefik#7096]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?
I launched Traefik in ECS Task in Fargate, with an attached task role.

### What did you expect to see?
Traefik use the metadata to retrieve `access_key`, `secret_access_key` and `token` from metadata.
I think Traefik doesn't use the good metadata endpoint, as Fargate has a dynamic endpoint.

### What did you see instead?

```
time="2020-07-25T08:35:22Z" level=error msg="Provider connection error EC2MetadataRequestError: failed to get EC2 instance identity document\ncaused by: RequestError: send request failed\ncaused by: Get \"http://169.254.169.254/latest/dynamic/instance-identity/document\": dial tcp 169.254.169.254:80: connect: invalid argument, retrying in 280.071208ms" providerName=ecs
```

### Output of `traefik version`: (_What version of Traefik are you using?_)
2.3.0-rc2 (Picodon)

### What is your environment & configuration (arguments, toml, provider, platform, ...)?
```
traefik --providers.ecs.clusters traefik-demo --api.insecure --log-level DEBUG
```
 on Fargate ECS

**Comment by rtribotte** [comment]
Hi @teddy-ferdinand ,

Thanks for your interest in Traefik !

Could you try to set the credentials, as stated in [this part of the documentation](https://docs.traefik.io/v2.3/providers/ecs/#credentials) ?
At least, by setting up the region (eg: `--providers.ecs.region="us-east-1"`) you should not encounter this error anymore.

**Comment by sl-teddyferdinand** [comment]
Hello, indeed, setting credentials with both CLI parameters and environments vars works (that's how I bypassed this issue temporary).

But it's not a good practice to use access_keys in AWS, because it's easier to get access_keys than temp access keys created by roles and I've to manage secret lifecycle.

**Comment by sl-teddyferdinand** [comment]
> Hello, indeed, setting credentials with both CLI parameters and environments vars works (that's how I bypassed this issue temporary).
> 
> But it's not a good practice to use access_keys in AWS, because it's easier to get access_keys than temp access keys created by roles and I've to manage secret lifecycle.

Wrong account, but it's still me ^^'

**Comment by traefiker** [comment]
Closed by #7145.
